utils/test/easy_track_helpers.py

```diff
@@ -161,27 +165,192 @@ def check_add_and_remove_recipient_with_voting(registry, helpers, ldo_holder, da
+
+
+def check_and_remove_mev_boost_relay_with_voting(
+    mev_boost_allowed_list, mev_boost_relay, helpers, ldo_holder, dao_voting
+):
+    relays = mev_boost_allowed_list.get_relays()
+
+    assert mev_boost_relay in relays
+
+    # Remove MEV-Boost relay with voting
     call_script_items = [
         agent_forward(
             [
                 (
-                    registry.address,
-                    registry.removeRecipient.encode_input(recipient_candidate),
+                    mev_boost_allowed_list.address,
+                    mev_boost_allowed_list.remove_relay.encode_input(mev_boost_relay),
                 )
             ]
         )
     ]
-    vote_desc_items = ["Remove recipient"]
+    vote_desc_items = ["Remove MEV-Boost relay"]
     vote_items = bake_vote_items(vote_desc_items, call_script_items)
 
     vote_id = create_vote(vote_items, {"from": ldo_holder})[0]
 
-    helpers.execute_vote(
+    vote_tx = helpers.execute_vote(
         vote_id=vote_id,
         accounts=accounts,
         dao_voting=dao_voting,
     )
 
-    assert not registry.isRecipientAllowed(recipient_candidate)
-    assert len(registry.getAllowedRecipients()) == recipients_length_before, "Wrong whitelist length"
+    relays_after = mev_boost_allowed_list.get_relays()
+
+    assert mev_boost_relay not in relays_after
+    assert len(relays_after) == len(relays) - 1, "Wrong allowed list length"
+
+
+def create_and_enact_add_mev_boost_relay_motion(
+    easy_track,
+    trusted_caller,
+    mev_boost_allowed_list,
+    factory,
+    relay,
+    stranger,
+    helpers,
+    ldo_holder,
+    dao_voting,
+):
+    relays = mev_boost_allowed_list.get_relays()
+
+    # Check if there is enough space in the list
+    if len(relays) >= MEV_BOOST_ALLOWED_LIST_MAX_RELAY_COUNT:
+        check_and_remove_mev_boost_relay_with_voting(mev_boost_allowed_list, relays[0], helpers, ldo_holder, dao_voting)
+
+    relays_count = len(relays)
+    assert relay not in relays
+
+    calldata = "0x" + encode(["(string,string,bool,string)[]"], [[relay]]).hex()
+
+    create_and_enact_motion(easy_track, trusted_caller, factory, calldata, stranger)
+
+    assert len(mev_boost_allowed_list.get_relays()) == relays_count + 1
+    assert relay in mev_boost_allowed_list.get_relays()
+
+
+def create_and_enact_remove_mev_boost_relay_motion(
+    easy_track,
+    trusted_caller,
+    mev_boost_allowed_list,
+    factory,
+    relay_uri,
+    stranger,
+    helpers,
+    ldo_holder,
+    dao_voting,
+):
+    relays = mev_boost_allowed_list.get_relays()
+
+    # Check if the relay is already in the list by the URI, but not by the whole tuple because of the edited fields
+    if relay_uri not in [x[0] for x in relays]:
+        new_relay = (relay_uri, "Lorem Ipsum Operator", True, "Description of the relay")
+        check_and_add_mev_boost_relay_with_voting(mev_boost_allowed_list, new_relay, helpers, ldo_holder, dao_voting)
```

Comment:
The `relay_uri` must always be equal to "https://0xaaccee.example.com", otherwise this line would throw an assert error? Probably it is not needed to pass the value for `relay_uri` as function arg, rather use the const from `TEST_RELAY`?